assets/scss/lib/_variables.scss

```diff
@@ -224,7 +224,7 @@ $navbar-dark-color: $white;
 $navbar-dark-hover-color: $yellow;
 $navbar-dark-active-color: $gray;
 $navbar-dark-disabled-color: rgba($white, .8);
-$navbar-dark-toggler-icon-bg: $gray;
+$navbar-dark-toggler-icon-bg: #{$gray};
```

Comment:
It took me a while to figure out what's happening here :)  After sometime spent in the [Sass documentation](https://sass-lang.com/documentation/interpolation#quoted-strings), it appears to be the equivalent of ```$navbar-dark-toggler-icon-bg: quote($gray);```.  Is that correct Paul?

Reply:
@Adnan-cds Yeah, I should've posted the error but in this file we're overriding bootstrap vars and this one in particular has to be a string otherwise it won't compile.